packages/story-editor/src/components/panels/design/alignment/useAlignment.js

```diff
@@ -79,12 +80,20 @@ function useAlignment({ selectedElements, updateElements }) {
       }),
     [selectedElements]
   );
-
   // Set boundRect with pageSize when there is only one element selected
   const boundRect =
     selectedElements.length === 1 ? PAGE_RECT : getBoundRect(selectedElements);
 
+  const handleTrackEvent = (direction) => {
+    trackEvent(isFloatingMenu ? 'floating_menu' : 'design_panel', {
+      name: `set_alignment_${direction}`,
+      element:
+        selectedElements.length > 1 ? 'multiple' : selectedElements[0]?.type,
```

Comment:
Can you align when only one element is selected? 🧐

Reply:
good catch. I don't think so. If anything tho, we could check if all the elements are the same type and pass that here? or just always do multiple?

Reply:
i have 'multiple' set up for the dismiss on floating so i'll just leave it as 'multiple'. great eye, thank you!! I saw the logic above checking for multiple so I just assumed it was an option somewhere i wasn't seeing.